### PR TITLE
Update outdated method prototype('array') with arrayPrototype() @se…

### PR DESCRIPTION
**Pull request by jafaronly**
…e https://github.com/symfony/symfony/issues/22505

| Q               | A
| --------------- | -----
| Branch?         | 1.0
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | yes 
| Related tickets | fixes 
| License         | MIT

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,5 @@
 /phpunit.xml
 /phpspec.yml
 /.web-server-pid
+
+/.idea/*
```
Comment (lchrusciel) [inline .gitignore]: It should not be a part of this PR. Please, add this folder to your global gitignore

File: src/Sylius/Bundle/GridBundle/Doctrine/ORM/ExpressionBuilder.php [hunk 2]
```diff
@@ -194,7 +194,7 @@ public function addOrderBy(string $field, string $direction)
     private function getFieldName(string $field): string
     {
         if (false === strpos($field, '.')) {
-            return $this->queryBuilder->getRootAlias() . '.' . $field;
+            return $this->queryBuilder->getRootAliases()[0] . '.' . $field;
```
Comment (lchrusciel) [inline src/Sylius/Bundle/GridBundle/Doctrine/ORM/ExpressionBuilder.php]: It should also be a part of separate PR according to PR name

**Comment by jafaronly** [comment]
@lchrusciel is it okay now ?

**Comment by pamil** [comment]
Thanks, Jafar! 🎉
